### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by jorshi**
Related to API changes made in https://github.com/neuralaudio/hear2021-eval-kit/pull/1

### CODE REVIEW

**Comment by turian** [comment]
We should include all these into the next push: https://github.com/neuralaudio/hear2021-eval-kit/issues?q=+label%3Aapi-update+

**Review by jorshi — COMMENTED** [review]
I think I've addressed most of the changes -- although I haven't added this into the API https://github.com/neuralaudio/hear2021-eval-kit/issues/49

File: docs/hear2021-holistic-evaluation-of-audio-representations.md [hunk 1]
```diff
@@ -120,31 +120,39 @@ the
 **Easy-to-use:**
 * Your code must use PyTorch >= 1.7 or Tensorflow >= 2.0. Notable marks will be given to models that work nearly identically for both libraries.
 * Speed:
-  * There will be two challenge tracks: 10x and 100x realtime.
-  * Your model must be fast enough to be able to process audio, which has already been loaded onto the GPU, at least 10x or 100x faster than realtime on a V100 GPU.
-  * If your model has more than one possible embedding size (see below for more information on embedding sizes), it must still output all embeddings together at the target speed.
+  * Your model must be fast enough to be able to process audio, which has already been loaded onto the GPU, and return at least X embeddings per second.
+  * Your model must be able to run at this speed using the following hardware architecture: X
+    * This is *not* the architecture that we will be running evaluations on.
```
[the remark this answers is not in the document]
Comment (turian) [reply docs/hear2021-holistic-evaluation-of-audio-representations.md:131]: Let's say 1000 frames per second on a GCP P100, V100, and A100.

File: docs/hear2021-holistic-evaluation-of-audio-representations.md [hunk 2]
```diff
@@ -120,31 +120,39 @@ the
...
 * Your model must be able to work on an 8GB GPU machine.
 
 <p></p>
 **Common format:**
 * Your code must follow a common API, described in the section below.
-* Your model must accept audio time series data of arbitrary length, as both a native tensor (perhaps already on CUDA) in the library of your choice, as well as numpy.ndarrays.
+* Your model must accept audio time series data of arbitrary length, as both a native tensor (perhaps already on CUDA) in the library of your choice, as well as *numpy.ndarrays*.
```
[the remark this answers is not in the document]
Comment (turian) [reply docs/hear2021-holistic-evaluation-of-audio-representations.md:137]: Okay

**Comment by jorshi** [comment]
This is old now - refer to #10

**Pull request closed without merging**